api/AudioContext.json

```diff
@@ -0,0 +1,552 @@
+{
+  "api": {
+    "AudioContext": {
+      "AudioContext": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/AudioContext/AudioContext",
+          "support": {
+            "chrome": [
+              {
+                "version_added": "60",
+                "notes": "Support for <code>options</code> parameter added."
+              },
+              {
+                "version_added": "55",
+                "version_removed": "59"
+              }
+            ],
+            "edge": {
+              "version_added": null
+            },
+            "firefox": [
+              {
+                "version_added": false
+              },
+              {
+                "version_added": "31",
+                "version_removed": "54",
+                "notes": "Non-standard constructor implemented, which accepted as input a string identifying the purpose for which the channel was being used. This was removed as it was causing code that used the <code>options</code> parameter to throw an exception. Correct support will arrive in the future."
```

Comment:
"Correct support will arrive in the future" is not needed as there is a false.
In fact I would not make an entry for it, I would have a note below the "version_added": false starting with ("From Firefox 31 to Firefox 53 included, a non-standard constructor was implemented, …")